### ISSUE TITLE
If already logged in, set context if no new auth needed

### ISSUE THREAD

**Issue by mlbiam** [TremoloSecurity/kubectl-login#2]
if there's already a context for the host set, see if it exists and is valid then try to check auth.  if auth succeeds, just set the context.  if it fails trigger login

**Comment by gregswift** [comment]
in #3 we say it would be nice to not have to pass host if a context is already selected, which is handy if you never left the context, but your logged out.  Here we are asking that it would be nice for context switching to trigger auth.

Current behavior:

```
$ kubectl ctx context-name
Switched to context "context-name".
$ kubectl get pods
Unable to connect to the server: failed to refresh token: oauth2: cannot fetch token: 401 Unauthorized
Response: 
$ kubectl oulogin --host=login.url.com
```

Requested "ideal" behavior:
```
$ kubectl ctx context-name
Switched to context "context-name".
```

alternate and would be acceptable:
```
$ kubectl oulogin context-name
Switched to context "context-name".
```

**Comment by mlbiam** [comment]
fixed in 0.0.5.  Update by running `kubectl krew upgrade oulogin`
